pom.xml

```diff
@@ -18,6 +18,11 @@
   </properties>
 
   <dependencies>
+    <dependency>
```

Comment:
Needed? If so explain and say it's only for Scala

Reply:
Where to explain?